docs/design/design-patterns/non-functional-requirements-investigation-guide.md

```diff
@@ -0,0 +1,148 @@
+
+e.g.,
+
+* Parent1 *[Susan/susan@domain.com]*
+  * child1
+  * child2 *[John/john@domain.com]*
+    * grandchild1
+  * child3
+* Parent2 *[Sam/sam@domain.com]*
+  * child1
+  * child2
+
+In the preceding example, 'Susan' is responsible for `Parent1` and all of its descendants _except_ for `Parent1/child2` and `Parent1/child2/grandchild1`  (for which 'John' is the stakeholder).  'Sam' is responsible for the entirety of `Parent2` and all of its descendants.
+
+This approach permits the retention of the logical hierarchy of elements themselves while also flexibly interleaving the 'stakeholder' identifications within the hierarchy of topics if/when they may need to diverge due to e.g., CUSTOMER organizational nuances.
+
+***
+
+## Areas of Investigation
+
+* Enterprise Security
+  * Privacy
+    * PII
+    * HIPAA
+  * Encryption
+  * Data mobility
+    * at rest
+    * in motion
+    * in process/memory
+  * Key Management
+    * responsibility
+      * platform
+      * BYOK
+      * CMK
+  * INFOSEC regulations/standards
+    * e.g., FIPS-140-2
+      * Level 2
+      * Level 3
+    * ISO 27000 series
+    * NIST
+    * Other?
+  * Network security
+    * Physical/Logical traffic boundaries/flow topology
+      * Azure <-- --> On-prem
+      * Public <-- --> Azure
+      * VNET
+      * PIP
+      * Firewalls
+      * VPN
+      * ExpressRoute
+        * Topology
+        * Security
+    * Certificates
+      * Issuer
+        * CA
+        * Self-signed
+        * Rotation/expiry
+  * INFOSEC Incident Response
+    * Process
+    * People
+    * Responsibilities
+    * Systems
+    * Legal/Regulatory/Compliance
+* Enterprise AuthN/AuthZ
+  * Users
+  * Services
+  * Authorities/directories
+  * Mechanisms/handshakes
+    * Active Directory
+    * SAML
+    * OAuth
+    * Other?
+  * RBAC
+    * Perms inheritance model
+* Enterprise Monitoring/Operations
+  * Logging
+    * Operations
+    * Reporting
+    * Audit
+  * Monitoring
+    * Diagnostics/Alerts
+    * Operations
+  * HA/DR
+    * Redundancy
+    * Recovery/Mitigation
+  * Practices
+    * Principle of least-privilege
+    * Principle of separation-of-responsibilities
+* Other standard Enterprise technologies/practices
+  * Developer ecosystem
+    * Platform/OS
+      * Hardened
+      * Approved base images
+      * Image repository
+    * Tools, languages
+      * Approval process
+    * Code repositories
+      * Secrets management patterns
+        * Env var
+        * Config file(s)
+        * Secrets retrieval API
+    * Package manager source(s)
+      * Private
+      * Public
+      * Approved/Trusted
+    * CI/CD
+    * Artifact repositories
+* Production ecosystem
+  * Platform/OS
+    * Hardened
+    * Approved base images
+    * Image repository
+  * Deployment longevity/volatility
+    * Automation
+    * Reproducibility
+      * IaC
+      * Scripting
+      * Other???
```

Comment:
Nit: remove the `???` here and in other parts, I think Other is enough to encourage folks to add/ask abotu additional areas

Reply:
Done.